sdk/core/ts-http-runtime/src/util/base64.browser.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
This is not needed? I don't see the corresponding node version

Reply:
it gets imported by sha256.browser.js -- IIRC Timo has cleaned some of this up in a recent PR to core-util, we can port his changes after that PR goes in.